.travis.yml

```diff
@@ -18,17 +18,15 @@ jobs:
       services: docker
       script:
         - docker pull ${SWIFT_DOCKER_IMAGE}
-        - args="apt-get update
-            && apt-get -y install libssl-dev zlib1g-dev
-            && swift build -c debug
-            && swift build -c release
-            && swift test -c debug --sanitize=thread --enable-test-discovery"
+        - args="swift build --enable-test-discovery -c debug
+            && swift build --enable-test-discovery -c release
+            && swift test --enable-test-discovery -c debug --sanitize=thread"
         - docker run --rm -v $(pwd):/app -v ~/.ssh/known_hosts:/root/.ssh/known_hosts:ro -v $SSH_AUTH_SOCK:/ssh-agent -e SSH_AUTH_SOCK=/ssh-agent --workdir /app ${SWIFT_DOCKER_IMAGE} bash -c "${args}"
 
     - stage: test
       name: Unit Tests
       os: osx
-      osx_image: xcode11.3
+      osx_image: xcode11.4
```

Comment:
We will have to look for another solution to run macOS tests since Vapor 4 requires 10.15 and the 11.4 image runs 10.14. More information can be found here: https://travis-ci.community/t/xcode-11-4-image-uses-macos-version-that-is-unsupported-by-that-version-of-xcode/8111